nsiqcppstyle_checker.py

```diff
@@ -28,6 +28,8 @@
 
 import os
 import traceback
+from nsiqcppstyle_outputer import _consoleOutputer as _consoleOutputer
```

Comment:
`from nsiqcppstyle_outputer import _consoleOutputer as console` is better since all calls would be `console(level, text)`